### ISSUE TITLE
Add a button on Smart Dashboard to toggle on/off closed loop driving

### ISSUE THREAD

**Issue by kushagharahi** [PearceRobotics/rapid-react-2022#9]
• Create a branch called toggle-drive-straight
• have the button set `gyroDisabled` boolean to true/false in subsystems/Drive.java when toggled
• Commit changes to branch you created above
• Open Pull Request against the main branch to merge in your code
This ticket depends on #3 being done and merged into the main branch
